domains/serenity-kiritoapi.adssousag.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "KiritoGamesPlays",
+    "email": "arthursantossousa05@gmail.com"
+  },
+  "records": {
+    "CNAME": "p3-host.triniumhost.com"
+  },
```

Comment:
Not seeing the portfolio related / developer related. No preview provided either. Denied.